README.md

```diff
@@ -6,7 +6,7 @@
 ![GitHub Discussions](https://img.shields.io/github/discussions/pulumi/pulumi)
 [![NPM version](https://badge.fury.io/js/%40pulumi%2Fpulumi.svg)](https://npmjs.com/package/@pulumi/pulumi)
 [![Python version](https://badge.fury.io/py/pulumi.svg)](https://pypi.org/project/pulumi)
-[![NuGet version](https://badge.fury.io/nu/pulumi.svg)](https://badge.fury.io/nu/pulumi)
+[![NuGet version](https://d25lcipzij17d.cloudfront.net/badge.svg?id=nu&r=r&type=6e&v=3.45.0&x2=0)](https://badge.fury.io/nu/pulumi)
```

Comment:
This is just the resolved URL (for now) of https://badge.fury.io/nu/pulumi.svg, I don't think it will accurately track as the nuget version changes.

Reply:
> This is just the resolved URL (for now) of https://badge.fury.io/nu/pulumi.svg, I don't think it will accurately track as the nuget version changes.

Thanks for your kind feedback!